CompetitionTemplate/src/competition/subsystems/arm/arm_commands/LowerArm.java

```diff
@@ -0,0 +1,31 @@
+package competition.subsystems.arm.arm_commands;
+import competition.subsystems.arm.ArmSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class LowerArm extends BaseCommand {
+    ArmSubsystem armSubsystem;
+
+    public LowerArm(ArmSubsystem armSubsystem) {
+        this.armSubsystem = armSubsystem;
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        if (armSubsystem.isArmAtMaximumHeight()){
+            end();
```

Comment:
you don't need to call end yourself, instead you want to have 'isFinished' return True which will call end() for you automatically.